lib/src/protocols/jsd_line.dart

```diff
@@ -162,7 +162,7 @@ class JdsLine implements CustomProtocolLine {
   ///
   /// converts json string into DsCommand 
   /// dipending on the type stored in the json['type']
-  static DsCommand _dsCommandFromJson(String json) {
+  static DsCommand dsCommandFromJson(String json) {
```

Comment:
This method should be private.

Reply:
Made private in: 
https://github.com/a-givertzman/hmi_networking/pull/21/commits/658d9a087514a7e0ce0664634224366fb346d4e0#diff-28005bc94dafde5c7e1ca631accc9f21e0e3d195493bba8811d7b0a6f4684c5f